### ISSUE TITLE
Adjusting early game, default campaign events

### ISSUE THREAD

**Issue by FulcrumA** [AnarkisGaming/ATC-Feedback#140]
**Posting here an entry from Discord, as requested:**
Related to #139 

> in standard campaign, on standard difficulty I get both traders and attacks by wild beasts, infected or bandits far, far too often than I'd expect them to be. Please consider decreasing frequency of those events at least early game, when the faction's small, the base still is being built and in the middle of nowhere, no exploration parties were yet sent etc - so it shouldn't gather much of any attention. Perhaps in general events should be "weighted" depending on player's faction power/number of personnel/wealth.

**Comment by IvanBajlo** [comment]
yes this can be very random... sometimes you get several attacks in a row, sometimes bunch of traders... if only we saw everything that is coming at us on the map... hmmm ;-)

**Comment by SerialKicked** [comment]
Raids are _already_ weighted depending on your technology, survivor count, and time played. 

A ratio of roughly 1:1 good to bad event ratio is kept. No more than 2 good or bad events in a row, outside of a very low 5% chance for a second event to happen only a few hours after the first. Events are generated every 2 to 3 in-game days. (default settings).

Story Mode manually control event settings, though. The end of act 1 will spam you with raids until you decide to move on, and some other raids are hardcoded to happen at specific points in the story, for instance. Story Mode doesn't need a balance pass at this point, though.

First two events are always a recruitment and a trader. Assuming you built a trading zone. That's nearly a week worth of peace, which is enough to build a basic kill zone, and more often then not, get to the first military tech level.

If that's more your jam, underground bases (bunker inc sandbox mode for instance) don't generate attention until you decide to pierce the surface (or place a trading zone, or do anything on the surface, really), you won't get traders or new survivors, though.

> yes this can be very random... sometimes you get several attacks in a row, sometimes bunch of traders... if only we saw everything that is coming at us on the map... hmmm ;-)

And it has roughly a 1:1 ratio, unless you uncheck "balance events" in the event settings :) 

Hordes randomly coming across your base, and being raided by raiders from the bandit hideout don't count as events. Hence this feeling. 

I'm not deleting the artificial/event raids tho. Map generated ones are too predictable and easy to cheese for the player to be only relied upon. The frequency can be edited in difficulty settings.

**Comment by IvanBajlo** [comment]
hmm maybe a "flashing warning" for bandits coming in from hideout (your base about to be raided)? or if enemy gets close to base to give player a little more warning? (option to disable for experienced players)
